src/http/common/entities/key-query.ts

```diff
@@ -1,17 +1,39 @@
+import { BadRequestException } from '@nestjs/common';
 import { ApiProperty } from '@nestjs/swagger';
+import { Transform, Type } from 'class-transformer';
+import { IsInt, IsBoolean, IsOptional, Min } from 'class-validator';
+
+const toBoolean = (value): boolean => {
+  if (value === 'true') {
+    return true;
+  }
+
+  if (value == 'false') {
+    return false;
+  }
+
+  throw new BadRequestException(['used must be a boolean value']);
```

Comment:
The name of the property is hard-coded in the function in exception. Maybe do the parameter with property name?

Reply:
Agree

Reply:
fixed